scripts/bench_cpu_stt.py

```
@@ -0,0 +1,260 @@
+
+    results = []
+    for engine_name, model, device in CONFIGS:
+        if args.only and args.only not in f"{engine_name}/{model}":
+            continue
+        cmd = [sys.executable, __file__, "--engine", engine_name, "--model", model,
+               "--device", device, "--set", args.set, "--json"]
+        if args.cores:
+            cmd += ["--cores", str(args.cores)]
+        if args.threads:
+            cmd += ["--threads", str(args.threads)]
+        out = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)